### PR TITLE
chore: add description for presentation requests

### PR DESCRIPTION
**Pull request by iagormoraes**
[Ticket](https://trello.com/c/IS0liH1W/5654-add-explanation-text-at-top-of-request-defined-in-the-presentation-request-update-request-page-design)

## Summary
Added the new description for presentation requests, by updaing the request body for hasMatchingCredentials.

## Changes
- updated api-overview
- updated terminology

## Testing
Tested locally.

## Demo

Screenshot 2023-07-24 at 09 54 03
Screenshot 2023-07-24 at 09 54 18

## Checklist

- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas _to a borderline excessive amount_
- [ ] I have made any relevant corresponding changes to the documentation including the project readme.
- [x] I have run and tested the changes locally
- [ ] If it is a core feature, I have added an appropriate amount of unit tests.
- [ ] Any dependent changes have been merged and published in upstream projects

### CODE REVIEW

**Review by leo1994 — APPROVED** [review]

File: docs/api-overview.mdx [hunk 1]
```diff
@@ -95,7 +95,8 @@ The same ApiKey necessary to call /hasMatchingCredentials grants you and only yo
   email?: string, // user's email address; optional if phone is provided
   phone?: string, // user's phone number; optional if email is provided
   credentialRequests: CredentialRequest[], // a list of one or more CredentialRequest objects. Encodes which credentials are being asked for; not used if partnerUuid is provided
-  partnerUuid?: string // partner's uuid; optional
+  partnerUuid?: string, // partner's uuid; optional
+  description?: string // the description that goes to the consent page header, when is empty the default text will be used.
```
Comment (rsmets) [inline docs/api-overview.mdx:99]: Nearly certain that the `description` attribute was only added to credential requests, not presentation request object.

**Review by rsmets — CHANGES_REQUESTED** [review]

File: docs/api-overview.mdx (hunk 1 above)
Comment (jacobleesinger) [inline docs/api-overview.mdx:99]:
```suggestion
  description?: string // A description displayed in the consent page header. If no description is provided, a default value will be displayed.
```

File: docs/terminology.mdx [hunk 2]
```diff
@@ -91,6 +91,7 @@ The full details of the request object aren't that important or helpful to know,
   "updatedAt": "1673470082200",
   "email": "test@verified.inc",
   "phone": null,
+  "description": "You are being requested with the following information:",
```
Comment (jacobleesinger) [inline docs/terminology.mdx:94]:
```suggestion
  "description": "Please share the following information:",
```

**Review by jacobleesinger — CHANGES_REQUESTED** [review]

File: docs/api-overview.mdx (hunk 1 above)
Comment (jacobleesinger) [reply docs/api-overview.mdx:99]: No, this is for the request-level description.

**Review by rsmets — APPROVED** [review]
Let's be sure to have the core service released to prod before we publish these docs.

**Pull request merged**